shared/styles/style-guide.desktop.js

```diff
@@ -44,7 +44,8 @@ export const globalColors = {
 }
 
 export const globalResizing = {
-  login: {width: 700, height: 580},
+  // Add 25px to login for the OS menubar
+  login: {width: 700, height: 605},
```

Comment:
I'd  be very surprised that this has to be touched. Maybe signup and login
and different sizes in zeplin?

On Friday, March 11, 2016, Marco Munizaga <notifications@github.com> wrote:

> In shared/styles/style-guide.desktop.js
> <https://github.com/keybase/client/pull/2298#discussion_r55873875>:
>
> > @@ -44,7 +44,8 @@ export const globalColors = {
> >  }
> >
> >  export const globalResizing = {
> > -  login: {width: 700, height: 580},
> > +  // Add 25px to login for the OS menubar
> > +  login: {width: 700, height: 605},
>
> This is what setContentSize should account for. Let me look why that
> isn't working. We shouldn't have to account for the menubar ourselves.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/2298/files#r55873875>.
>